package.json

```diff
@@ -79,7 +79,7 @@
 		"del-cli": "^4.0.1",
 		"dts-bundle-generator": "^6.4.0",
 		"esbuild": "^0.14.12",
-		"express": "^4.17.1",
```

Comment:
oh, I think this should probably stay on 4.x if it's going to be the starting point for tests that first run against 4.x, then5.x

Reply:
This version is actually not changed during the tests - its only the version of express used by the projects that changes. This express dependency is only for type definitions, and AFAIK type defs for 5.0.0 haven't been released (or maybe they are the same as 4.x?)

Reply:
Perhaps this should stay at 4.17 then

Reply:
yeah, I guess it doesn't matter too much, but let's keep it at 4.x for now.